test/Storage/Adapter/AbstractAdapterTest.php

```diff
@@ -497,7 +497,7 @@ public function testEventHandlingStopInPre(
         $eventManager->attach($methodName . '.post', $eventHandler);
         $eventManager->attach($methodName . '.exception', $eventHandler);
 
-        $eventManager->attach($methodName . '.pre', static function ($event) use ($retVal) {
+        $eventManager->attach($methodName . '.pre', static function ($event) use ($retVal): mixed {
```

Comment:
If Psalm is forcing this return type addition, can we baseline this too please? Purely to reduce merge-up conflicts.

Reply:
Done with e099a96